examples/rust/components/http-task-manager/README.md

```diff
@@ -0,0 +1,116 @@
+# HTTP Task manager
+
+This web component tracks and updates information about asynchronous tasks, backed by a [Postgres][postgres] Database.
+
+Clients can make HTTP requests to endpoints like the following:
+
+| Endpoint                 | Description                   |
+|--------------------------|-------------------------------|
+| `GET /tasks`             | List ongoing tasks            |
+| `POST /tasks/:id/status` | Get status of a specific task |
+| `POST /tasks`            | Submit a new task             |
```

Comment:
do we want to note the `/api/v1` prefix here?

Reply:
Ah yes, thanks, will update here, all these endpoints are outdated actually (and ab bunch are missing)